test/integration/node-specific/mapreduce.test.js

```diff
@@ -14,7 +13,7 @@ describe('MapReduce', function () {
    */
   it('shouldPerformMapReduceWithStringFunctions', {
     metadata: {
-      requires: { topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'] }
+      requires: { topology: ['single', 'replicaset', 'sharded'] }
```

Comment:
I see we removed the extraneous ones here, any reason to not remove them all?

Reply:
My experience with removing the topology requirement in mongo client wasn't great, see my answer to Bailey's question; I mean we can try it here, but we may end up taking all day to figure out what is or isn't safe to remove